src/app/components/rss/sidebar-rss-feed/sidebar-rss-feed.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component } from '@angular/core';
+import { RssFeedService } from '../../../services/RssFeedService/rss.service';
+import { RssFeed } from '../../../models/RssFeed';
+import { RssResponse } from '../../../interface/RssResponse';
+
+@Component({
+  selector: 'app-sidebar-rss-feed',
+  templateUrl: './sidebar-rss-feed.component.html',
+  styleUrls: ['./sidebar-rss-feed.component.scss'],
+})
+export class SidebarRssFeedComponent {
+  isOpen = false;
+  rssLink!: string[];
+  rssData: any[] = [];
```

Comment:
each rssData has a different model ? if yes any is ok else create a pattern as interface or as model